### ISSUE TITLE
Suggestion: provide the 'simple_fruit_sales.csv' 

### ISSUE THREAD

**Issue by konstantinstadler** [ricklupton/floweaver#7]
The mybinder binding is great for experimenting. However, it would be quite useful to also be able to quickly run the quickstart locally. Could you provide the required csv file in the example folder?

**Comment by ricklupton** [comment]
Good idea, thanks. I guess I need to unite the newer "tutorials" and the older "examples".
